pkg/varlinkapi/images.go

```diff
@@ -508,7 +508,7 @@ func (i *LibpodAPI) PullImage(call iopodman.VarlinkCall, name string) error {
 // ImageExists returns bool as to whether the input image exists in local storage
 func (i *LibpodAPI) ImageExists(call iopodman.VarlinkCall, name string) error {
 	_, err := i.Runtime.ImageRuntime().NewFromLocal(name)
-	if errors.Cause(err) == libpod.ErrNoSuchImage {
+	if errors.Cause(err).Error() == libpod.ErrNoSuchImage.Error() {
```

Comment:
Woah - this didn't work without forcing a string comparison?

Reply:
Oh - it's image.ErrNoSuchImage

Reply:
(Where image is libpod/image)

Reply:
no, i think the error being returned has been been munged sufficiently?

Reply:
I think you can drop the .Error() if you swap to `errors.Cause(err) == image.ErrNoSuchImage`

Reply:
(We should probably drop `libpod.ErrNoSuchImage` to prevent this from happening int he future)